cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1057,9 +1057,7 @@ Slice::Gen::ForwardDeclVisitor::visitModuleEnd(const ModulePtr&)
 void
 Slice::Gen::ForwardDeclVisitor::visitClassDecl(const ClassDeclPtr& p)
 {
-    ClassDefPtr def = p->definition();
```

Comment:
More dead variables.
There's got to be some linter/tool for removing these things right?